presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5391,7 +5391,7 @@ public void testDuplicateFields()
     @Test
     public void testDuplicateColumnsInWindowOrderByClause()
     {
-        MaterializedResult actual = computeActual("SELECT a, row_number() OVER (ORDER BY a, a) FROM (VALUES 3, 2, 1) t(a)");
+        MaterializedResult actual = computeActual("SELECT a, row_number() OVER (ORDER BY a ASC, a DESC) FROM (VALUES 3, 2, 1) t(a)");
```

Comment:
Nice find!

Reply:
Can you please add similar tests for regular `ORDER BY`, any other place where `ORDER BY` can be used.

Reply:
Sure, I can, although this patch has nothing to do with `ORDER BY` out of the WF scope.